keps/sig-scheduling/2779-support-custom-queue-for-the-scheduler/README.md

```diff
@@ -84,6 +84,14 @@ Given that the business needs may vary greatly, it'd be desirable to provide a r
 implement their custom queue implementation, while the upstream maintainers focus
 on keeping the core small and extensible, and thus only maintain the internal queue piece.
 
+This KEP is not used to change the internal queue, or queue sorting plugin, quite the contrary,
```

Comment:
This should go as Non-Goal

Reply:
The above message is a little long for `Non-Goals`, how about updating the readme as following?

Add message `Enable scheduler developers to change the internal queue, or queue sorting plugin` in the `Non-Goals` section to show that it is not our goal.

Keep the orignal message in the `Motivation` section to let others know the details of our design.